test/test_intersect.py

```diff
@@ -0,0 +1,135 @@
+import pytest
+import os
+import numpy as np
+import pandas as pd
+import shutil
+import subprocess
+import xarray as xr
+import rioxarray as xrr
+import RAiDER
+import yaml
+import glob
+from test import TEST_DIR, WM
+wm = 'ERA-5' if WM == 'ERA5' else WM
+
+def makeLatLonGrid(bbox, reg, out_dir, spacing=0.1):
+    """ Make lat lons at a specified spacing """
+    S, N, W, E = bbox
+    lat_st, lat_en = S, N
+    lon_st, lon_en = W, E
+
+    lats = np.arange(lat_st, lat_en, spacing)
+    lons = np.arange(lon_st, lon_en, spacing)
+    Lat, Lon = np.meshgrid(lats, lons)
+    da_lat = xr.DataArray(Lat.T, name='data', coords={'lon': lons, 'lat': lats}, dims='lat lon'.split())
+    da_lon = xr.DataArray(Lon.T, name='data', coords={'lon': lons, 'lat': lats}, dims='lat lon'.split())
+
+    dst_lat = os.path.join(out_dir, f'lat_{reg}.nc')
+    dst_lon = os.path.join(out_dir, f'lon_{reg}.nc')
+    da_lat.to_netcdf(dst_lat)
+    da_lon.to_netcdf(dst_lon)
+
+    return dst_lat, dst_lon
+
+
+def update_yaml(dct_cfg:dict, dst:str='temp.yaml'):
+    """ Write a new yaml file from a dictionary.
+
+    Updates parameters in the default 'raider.yaml' file.
+    Each key:value pair will in 'dct_cfg' will overwrite that in the default
+    """
+
+    template_file = os.path.join(
+                    os.path.dirname(RAiDER.__file__), 'cli', 'raider.yaml')
+
+    with open(template_file, 'r') as f:
+        try:
+            params = yaml.safe_load(f)
+        except yaml.YAMLError as exc:
+            print(exc)
+            raise ValueError(f'Something is wrong with the yaml file {template_file}')
+
+    params = {**params, **dct_cfg}
+
+    with open(dst, 'w') as fh:
+        yaml.safe_dump(params, fh,  default_flow_style=False)
+
+    return dst
+
+
+def test_cube_intersect():
+    """ Test the intersection of lat/lon files with the DEM (model height levels?) """
+    TEST_DIR = '/Users/buzzanga/Software_InSAR/RAiDER_git/test'
```

Comment:
This won't work on circleci. :) You can import the TEST_DIR I think.

Reply:
You're right of course, but somehow the test passed?